feconf.py

```diff
@@ -77,7 +77,7 @@
         'current_data_schema_version': 1
     }
 }
-
+TTL = 5*5*60
```

Comment:
A more suitable name for this and comment. You can name it `CLASSIFIER_JOB_TTL` or something which relates with its usage.